ocrd_tool.md

```diff
@@ -10,6 +10,84 @@ services](swagger).
 
 To validate a `ocrd-tool.json` file, use `ocrd ocrd-tool /path/to/ocrd-tool.json validate`.
 
+## Standard parameters
+
+There is a number of parameters common to all processors that MUST be supported by processors.
+
+### `dpi`
+
+Custom DPI to assume for pixel density of images.
```

Comment:
I have already [proposed](https://github.com/OCR-D/core/issues/376#issuecomment-567390606) a better solution for DPI overrides: by static annotation in the respective PAGE-XML attributes. `OcrdExif` would then have to prefer that over the binary meta-data if available. And dedicated processors could estimate DPI from the image, or allow user overrides. I thought you were going along with that?

Reply:
I do. I'd still offer it as a standard parameter with `parameter > annotation > EXIF metadata`.

Reply:
Ah, I see. Makes sense (as long as we make sure these standard parameters are really automatic for tools).